superset/views/sql_lab.py

```diff
@@ -15,16 +15,38 @@
 # specific language governing permissions and limitations
 # under the License.
 # pylint: disable=C,R,W
+from typing import Callable
+
 from flask import g, redirect
 from flask_appbuilder import expose
 from flask_appbuilder.models.sqla.interface import SQLAInterface
 from flask_appbuilder.security.decorators import has_access
 from flask_babel import gettext as __
 from flask_babel import lazy_gettext as _
+import flask_sqlalchemy
```

Comment:
You may want to import this as `from flask_sqlalchemy import BaseQuery` which will reduce repetition below.